OSDOCS-1769 Installing a cluster on AWS in a restricted network (#31935)
openshift/openshift-docs: modules/installation-initializing.adoc

----
+
The value must be the contents of the certificate file that you used for your mirror registry, which can be an existing, trusted certificate authority or the self-signed certificate that you generated for the mirror registry.
ifdef::aws+restricted[]

> .. Define the subnets for the VPC your mirror registry is hosted on:

Reviewer: I would omit the reference to the mirror registry.
```suggestion
.. Define the subnets for the VPC to install the cluster in:
```
